packages/react/src/primitives/types/theme.ts

```diff
@@ -0,0 +1,183 @@
+/**
+ * Token keys for colors
+ *
+ * Though both 'white' and 'black' are valid CSS values
+ * We still want to define them as keys here and convert them into CSS variable reference afterwards
+ * This will make the default dark mode override work, where we flip these two
+ */
+type WhiteKey = 'white';
+
+type BlackKey = 'black';
+
+type RedKeys =
```

Comment:
These are for type intellisense. No worries, they are no more than syntactic sugar in Typescript and will not increase complied javascript size.